app/tests.py

```diff
@@ -68,4 +68,31 @@ def test_login_required_on_IndexPageView(self):
         request = self.factory.get(reverse('app:index'))
         request.user = self.user
         response = view(request)
-        self.assertEqual(response.status_code, 200)
\ No newline at end of file
+        self.assertEqual(response.status_code, 200)
+
+
+    def test_question_exists_on_questions_page(self):
+        view = IndexPageView.as_view()
+
+        request = self.factory.get(reverse('app:index'))
```

Comment:
Use the client instead of the factory so we can test full end to end of the route.